homework/Metaclass with inheritance.py

```diff
@@ -0,0 +1,20 @@
+# 3. Create metaclass with inheritance
+class MetaCheburek(type):
+    def __new__(mcs, class_name, *args, **kwargs):
+        print("We are created Cheburek class object")
+        class_obj = super().__new__(mcs, class_name, *args, **kwargs)
+        class_obj.ingredients = ['Potato, Cheese, Salad']
+        return class_obj
+
+
+class food:
```

Comment:
name conversation for the classes should start from the upper letter, like `Food`

Reply:
fixed